src/styles/index.scss

```diff
@@ -1,3 +1,248 @@
+$primary-color: #00acdc;
+$font-color: #060b35;
+$light-grey: #616070;
+$font-family: 'Roboto', sans-serif;
+
+:root {
+  --blue-color: #00acdc;
+  --black-color: #060b35;
+}
+
+html {
+  margin: 0;
+  min-width: 1024px;
+  font-family: $font-family;
+  font-size: 12px;
+  line-height: 14.06px;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: end;
+  gap: 243px;
```

Comment:
redundant

Reply:
What exactly can be simplified here?

Reply:
<img width="594" alt="image" src="https://github.com/user-attachments/assets/55e4f256-e57c-4d50-829f-d5714f228f2f" />
remove the gap and change the justify-content to space-between